### ISSUE TITLE
Multple record identifying fields cause a BeanIOConfigurationException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#13]
```
Since release 1.0, multiple record identifying fields (i.e. rid="true") in the 
same record cause a BeanIOConfigurationException with the error message "a 
collection cannot contain record identifying fields".

This will be fixed in the next release.
```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 23 Jul 2011 at 5:52

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 1.2.0.
```

Original comment by `kevin.s...@gmail.com` on 6 Sep 2011 at 11:28
- Added labels: **Milestone-1.2.0**

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 9 Sep 2011 at 1:40
- Changed state: **Fixed**
